nix/shell.sh

```diff
@@ -23,6 +23,8 @@ libcrypto_alias openssl102 "${OPENSSL_1_0_2_INSTALL_DIR}/bin/openssl"
 libcrypto_alias openssl111 "${OPENSSL_1_1_1_INSTALL_DIR}/bin/openssl"
 libcrypto_alias openssl30 "${OPENSSL_3_0_INSTALL_DIR}/bin/openssl"
 libcrypto_alias bssl "${AWSLC_INSTALL_DIR}/bin/bssl"
+libcrypto_alias fips2022bssl "${AWSLC_FIPS_2022_INSTALL_DIR}/bin/bssl"
+libcrypto_alias fips2024bssl "${AWSLC_FIPS_2024_INSTALL_DIR}/bin/bssl"
```

Comment:
Nit unrelated to this PR, but should we be referring to all of these as bssl? Like, libressl isn't call openssl just bc that's the binary name. Shouldn't these binaries all by awslc? We also do support plain boringssl, which is probably the real "bssl" :P

Reply:
What's even more fun is that aws-lc ships an `openssl` binary now as well...it's a fair point though, these are just aliases, so we can make them sensible.